contracts/modules/SuperMinterV1_1.sol

```diff
@@ -1040,21 +1076,69 @@ contract SuperMinterV1_1 is ISuperMinterV1_1, EIP712 {
+        unchecked {
+            uint256 mintId = LibOps.packId(p.edition, p.tier, p.scheduleNum);
+            uint256 toLength = p.to.length;
+
+            // Increment the number minted in the mint.
+            uint256 n = uint256(d.minted) + toLength * uint256(p.signedQuantity); // The next `minted`.
+            if (n > d.maxMintable) revert ExceedsMintSupply();
+            d.minted = uint32(n);
+
+            // Increment the number minted by the collectors.
+            for (uint256 i; i != toLength; ++i) {
+                LibMap.Uint32Map storage m = _numberMinted[p.to[i]];
+                m.set(mintId, uint32(uint256(m.get(mintId)) + uint256(p.signedQuantity)));
+            }
+        }
+    }
+
+    /**
+     * @dev Requires that the mint is open and not paused.
+     * @param d    The mint data storage pointer.
+     */
+    function _requireMintOpen(MintData storage d) internal view {
+        if (LibOps.or(block.timestamp < d.startTime, block.timestamp > d.endTime))
+            revert MintNotOpen(block.timestamp, d.startTime, d.endTime);
+        if (_isPaused(d)) revert MintPaused(); // Check if the mint is not paused.
+    }
+
     /**
      * @dev Verify the signature, and mark the signed claim ticket as claimed.
      * @param d The mint data storage pointer.
      * @param p The mint-to parameters.
      */
     function _verifyAndClaimSignature(MintData storage d, MintTo calldata p) internal {
         if (p.quantity > p.signedQuantity) revert ExceedsSignedQuantity();
-        address signer = _effectiveSigner(d);
+        address signer = platformSigner[d.platform];
         if (!SignatureCheckerLib.isValidSignatureNowCalldata(signer, computeMintToDigest(p), p.signature))
             revert InvalidSignature();
         if (block.timestamp > p.signedDeadline) revert SignatureExpired();
         uint256 mintId = LibOps.packId(p.edition, p.tier, p.scheduleNum);
         if (!_claimsBitmaps[mintId].toggle(p.signedClaimTicket)) revert SignatureAlreadyUsed();
     }
 
+    /**
+     * @dev Verify the platform airdrop signature, and mark the signed claim ticket as claimed.
+     * @param d The mint data storage pointer.
+     * @param p The platform airdrop parameters.
+     */
+    function _verifyAndClaimPlatfromAidropSignature(MintData storage d, PlatformAirdrop calldata p) internal {
+        // Unlike regular signature mints, platform airdrops only use `signedQuantity`.
+        address signer = platformSigner[d.platform];
+        if (!SignatureCheckerLib.isValidSignatureNowCalldata(signer, computePlatformAirdropDigest(p), p.signature))
+            revert InvalidSignature();
+        if (block.timestamp > p.signedDeadline) revert SignatureExpired();
+        uint256 mintId = LibOps.packId(p.edition, p.tier, p.scheduleNum);
+        if (!_claimsBitmaps[mintId].toggle(p.signedClaimTicket)) revert SignatureAlreadyUsed();
```

Comment:
So we have the EIP712 signature verification that uses the entire PlatformAirdrop object (except the signature). Then we verify the expiry, then check if it has been used before. Checked toggle to make sure that when the bit is unset it returns true, and that seems to be the case. Reversion should keep toggle from turning the bit off. Should work as intended.